TA_Scheduler_App/views.py

```diff
@@ -270,3 +271,50 @@ def post(self, request, course_id):
         return redirect("courses")
 
 
+
+class Assignments(View):
+    def get(self,request):
+        courses = Course.objects.all()
+        relations = teacherToTA.objects.all().filter(teacherIDF=request.user)
+        return render(request, 'courses.html',{'relations':relations,'courses':courses})
+
+    def post(self,request, section=None,assignment=None):
+        action = request.POST.get('action')
+        if action == 'create':
+            assignment_features.create_assignment(self,section,request.user)
+            return redirect('courses')
+        if action == 'delete':
+            if assignment is not None:
+                assignment_features.delete_assignment(self,assignment.pk)
+
+        return redirect('course')
+
+
+
+class Skills(View):
+    def get(self,request):
+        skill = Skills.objects.all().filter(userId=request.user.pk)
+        return render(request, 'accounts.html', {'skills':skill})
+
+    def post(self,request,skillId=None,skillString=None):
```

Comment:
```suggestion
    def post(self, request, skillId=None, skillString=None):
```
`skillId` as a name is awful for readability but that'll get fixed with the big reformatting pass